### PR TITLE
Nuke gulp and guides

### PR DESCRIPTION
**Pull request by ProstoSanja**
title says it all

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -13,39 +13,60 @@ Install the Dota 2 Workshop Tools.
...
 ```
-d2am list
+npm install dota2-addon-manager
 ```
-leave the command prompt open, we'll come back to it. Join us on iscordDay, the URL ends in WNFBB4d. You found the easter egg!
 
-## Get The Addon
-Use the Github client to check out this addon into the addons directory. You can use the "Clone or Download" button right on the main page to open it using the desktop tool. Make sure it ends up in addons, the path should end in `dota 2 beta\addons\oaa\`.
+### Download Project
+This is simple, you have to open `GitHub Client` and using top left `+` button `colne` a repo with `OAA` to your newly created `Open Angel arena folder`
```
Comment (warpdragon) [inline docs/install.md]: 'colne' should be 'clone'

File: docs/testing.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Testing Open Angel Arena
+This is the continuation of [Install guide](/docs/install.md)
+Here is a guide how you can modify and run the OAA
+
+## Preparation
+You first need to open dev tools and load OAA project
```
Comment (warpdragon) [inline docs/testing.md]: Should read:
"You first need to open 'Dota 2 Tools' and load OAA project". There is no such program called "Dev Tools", right? It's Dota 2 Tools.

File: docs/testing.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Building a map
+For game to run you also need a map,
+It is rebuilt by double-clicking `aaa.vmap` in `Asset browser` and then pressing `f9` and secting `Full compile` at top and `Build` at bottom of dialogue.
+`It should be done evey time you make changes to the map`
+
+## Running a game
+Now when you are set, you can start debugging your game, for this you need to open `dota console` in `Asset Browser`(Button on top)
```
Comment (warpdragon) [inline docs/testing.md]: mention that the default key for opening console is "~"

File: docs/testing.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+```
+dota_launch_custom_game (mod name) (map name)
+dota_launch_custom_game oaa aaa
+```
+after this dota client will start running a gamemode
+
+## Debugging
+To debug a game or any `panorama ui` you can use that `dota console` and commands `pause` and `unpause`
+For debugging a `panorama ui`(This new blue-grey design, aka reborn, aka 7.00) you at any time can press `f6` ingame and click on objects to inspect them in panorama debugger, which also features `javascript console`
+Ingame cheats, as -gols and -lvlup also work for any testing purposes
```
Comment (warpdragon) [inline docs/testing.md]: "-gols" should be "-gold"

File: docs/testing.md (hunk 3 above)
Comment (Chronophylos) [inline docs/testing.md]: Uhm for me its `^`. You should say its the Button under `ESC`, left of `1` and above of `Tab`.

**Comment by ProstoSanja** [comment]
@Chronophylos now its okay for you?

**Comment by Chronophylos** [comment]
You are still using the old files. They have changed. AAA was changed to OAA and so on. You need to merge the master into your branch.

**Comment by ProstoSanja** [comment]
@Chronophylos I just did, look at it, it is 3 ahead and 0 behind. The map wasn't changed to oaa, everything other was. Be more attentive please.

File: README.md [hunk 5]
```diff
@@ -4,23 +4,22 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
-For detailed instructions, check [here](docs/install.md)
+For installation instructions click [here](docs/install.md).
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
 Check out the Discord!
 
-There are also issues marked with [`Great for new contributors`](https://github.com/OpenAngelArena/oaa/issues?q=is%3Aissue+is%3Aopen+label%3A%22great+for+new+contributor%22).
+There are also issues marked with [`Great for new contributors`](https://github.com/AngelArenaAllstars/aaa/issues?q=is%3Aissue+is%3Aopen+label%3A%22great+for+new+contributor%22).
```
Comment (Chronophylos) [inline README.md]: oaa OpenAngelArena

File: README.md [hunk 6]
```diff
@@ -4,23 +4,22 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
 
-### Building npc custom files
-From within the addon directory, run `gulp`.
-
 # License
 MIT
 
-[soundcloud-link]: https://soundcloud.com/OpenAngelArena "Music for Open Angel Arena"
-[discord-link]: https://discord.gg/WNFBB4d "Open Angel Arena Discord Instant Invite"
+[soundcloud-link]: https://soundcloud.com/angelarenaallstars "Music for Angel Arena Allstars"
```
Comment (Chronophylos) [inline README.md]: OpenAngelArena Open Angel Arena

File: README.md [hunk 7]
```diff
@@ -4,23 +4,22 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
+[discord-link]: https://discord.gg/ "Angel Arena Allstars Discord Instant Invite"
```
Comment (Chronophylos) [inline README.md]: Open Angel Arena

File: docs/install.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
-# Installing Open Angel Arena
-You have to set up several things before you can run a local instance of AAA for development.
+# Installing Angel Arena Allstars
```
Comment (Chronophylos) [inline docs/install.md]: Open Angel Arena

**Comment by ProstoSanja** [comment]
Then be more correct, I was not "using old files" these were "old links""

**Comment by Chronophylos** [comment]
You are overriding `OAA` `OpenAngelArena` and `Open Angel Arena` on several occasions with `AAA` `AngelArenaAllstars` and `Angel Arena Allstars`.

File: README.md [hunk 9]
```diff
@@ -4,23 +4,19 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
 This addon uses the [Dota 2 Addon Manager](https://github.com/chrisinajar/dota2-addon-manager).
 
-For detailed instructions, check [here](docs/install.md)
+For installation instructions click [here](docs/install.md).
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
-Check out the Discord!
+Check out the Discord
```
Comment (Chronophylos) [inline README.md]: You can link the discord here.

**Comment by ProstoSanja** [comment]
Done

File: README.md [hunk 10]
```diff
@@ -4,23 +4,19 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
 
 There are also issues marked with [`Great for new contributors`](https://github.com/OpenAngelArena/oaa/issues?q=is%3Aissue+is%3Aopen+label%3A%22great+for+new+contributor%22).
 
-Music can be found at [Soundcloud][soundcloud-link]
```
Comment (Chronophylos) [inline README.md]: Don't remove the soundcloud link.

Comment (ProstoSanja) [inline README.md]: Give me the correct one, I can't find

Comment (ProstoSanja) [inline README.md]: Wait, I left it there, look below

**Comment by ProstoSanja** [comment]
Finally done

**Comment by ProstoSanja** [comment]
Can we never this?

**Comment by ProstoSanja** [comment]
Requesting tenure by @Chronophylos

File: README.md [hunk 11]
```diff
@@ -4,23 +4,19 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
 This addon uses the [Dota 2 Addon Manager](https://github.com/chrisinajar/dota2-addon-manager).
 
-For detailed instructions, check [here](docs/install.md)
+For installation instructions click [here](docs/install.md).
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
-Check out the Discord!
+Check out our [Discord](https://discord.gg/E6f6TCe)!
```
Comment (Chronophylos) [inline README.md]: Thats the wrong discord. WNFBB4d

**Comment by Chronophylos** [comment]
What does "requesting tenure" mean?

File: README.md (hunk 11 above)
Comment (ProstoSanja) [inline README.md]: That's also link, to this server, just created by me, also permanent/general/oaa but generated from my account

Tenure - review (t9 from phone)

**Comment by ProstoSanja** [comment]
Done

**Comment by ProstoSanja** [comment]
also finished with this, @Chronophylos please give final review

**Comment by Chronophylos** [comment]
Will do final one when I'm home.

File: README.md [hunk 12]
```diff
@@ -4,23 +4,19 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
-Check out the Discord!
+Check out our [Discord](https://discord.gg/WNFBB4d)!
 
 There are also issues marked with [`Great for new contributors`](https://github.com/OpenAngelArena/oaa/issues?q=is%3Aissue+is%3Aopen+label%3A%22great+for+new+contributor%22).
 
-Music can be found at [Soundcloud][soundcloud-link]
```
Comment (Chronophylos) [inline README.md]: Leave the Soundcloud link in but use `https://soundcloud.com/OpenAngelArena`

File: docs/install.md [hunk 13]
```diff
@@ -13,39 +13,60 @@ Install the Dota 2 Workshop Tools.
...
 ```
-d2am list
+npm install dota2-addon-manager
 ```
-leave the command prompt open, we'll come back to it. Join us on iscordDay, the URL ends in WNFBB4d. You found the easter egg!
 
-## Get The Addon
-Use the Github client to check out this addon into the addons directory. You can use the "Clone or Download" button right on the main page to open it using the desktop tool. Make sure it ends up in addons, the path should end in `dota 2 beta\addons\oaa\`.
+### Download Project
+This is simple, you have to open `GitHub Client` and using top left `+` button `clone` a repo with `OAA` to your newly created `Open Angel arena folder`
```
Comment (Chronophylos) [inline docs/install.md]: You should reference the git guide here.

File: docs/testing.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+## Building a map
+For game to run you also need a map,
+It is rebuilt by double-clicking `aaa.vmap` in `Asset browser` and then pressing `f9` and secting `Full compile` at top and `Build` at bottom of dialogue.
+`It should be done evey time you make changes to the map`
+
+## Running a game
+Now when you are set, you can start debugging your game, for this you need to open `dota console` in `Asset Browser`(Button on top or a key `~` [located under `esc` and above `tab`])
+Then a command to launch game is:
+```
+dota_launch_custom_game (mod name) (map name)
```
Comment (Chronophylos) [inline docs/testing.md]: Use `dota_launch_custom_game <mod name> <map name>`. `<>` are used for required arguments.

File: docs/install.md [hunk 15]
```diff
@@ -6,7 +6,7 @@ Start by installing these programs:
  * [Steam](http://steampowered.com)
  * [nodejs](http://nodejs.org)
  * [Github client](http://desktop.github.com/)
- * [Sublime Text 3](http://www.sublimetext.com/) **optional**
```
Comment (Chronophylos) [inline docs/install.md]: Don't remove ST3. A lot of people are using it.
And if you're at it add Atom Editor as well.

File: docs/install.md [hunk 16]
```diff
@@ -1,51 +1,73 @@
 # Installing Open Angel Arena
-You have to set up several things before you can run a local instance of AAA for development.
+You have to set up several things before you can run a local instance of OAA for development.
 
 ## System Dependencies
 Start by installing these programs:
  * [Steam](http://steampowered.com)
  * [nodejs](http://nodejs.org)
  * [Github client](http://desktop.github.com/)
- * [Sublime Text 3](http://www.sublimetext.com/) **optional**
```
Comment (Chronophylos) [inline docs/install.md]: Don't remove ST3. A lot of people are using it.
And if you're at it add Atom Editor as well.

File: docs/install.md [hunk 17]
```diff
@@ -1,51 +1,73 @@
...
+ * [Visual Studio Code](http://code.visualstudio.com/) `Text editor which includes support for github, editorconfig and much more`
 
 Install the Dota 2 Workshop Tools.
  * From within steam, right click on `Dota 2` and select `Properties` at the bottom
  * Go to the DLC tab of the window that pops up
  * Check the `INSTALL` box next to `Dota 2 Workshop Tools DLC`
 
-## Addon Tool
+## Setting evrything up
```
Comment (Chronophylos) [inline docs/install.md]: everything*

File: README.md [hunk 18]
```diff
@@ -4,23 +4,21 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
 This addon uses the [Dota 2 Addon Manager](https://github.com/chrisinajar/dota2-addon-manager).
 
-For detailed instructions, check [here](docs/install.md)
+For installation instructions click [here](docs/install.md).
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
-Check out the Discord!
+Check out our [Discord](https://discord.gg/WNFBB4d)!
```
Comment (chrisinajar) [inline README.md]: why remove the url variable? also i'm not sure if we want this on the front page yet.  Also this has nothing to do with gulp.

File: README.md [hunk 19]
```diff
@@ -4,23 +4,21 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
...
 
-### Building npc custom files
-From within the addon directory, run `gulp`.
-
 # License
 MIT
 
-[soundcloud-link]: https://soundcloud.com/OpenAngelArena "Music for Open Angel Arena"
-[discord-link]: https://discord.gg/WNFBB4d "Open Angel Arena Discord Instant Invite"
+[Soundcloud](https://soundcloud.com/OpenAngelArena)
```
Comment (chrisinajar) [inline README.md]: why rename the variable? this seems like a pointless change that makes this document less standard with the other documents. Also this has nothing to do with gulp.

File: docs/install.md [hunk 20]
```diff
@@ -1,51 +1,73 @@
...
-You have to set up several things before you can run a local instance of AAA for development.
+You have to set up several things before you can run a local instance of OAA for development.
 
 ## System Dependencies
 Start by installing these programs:
  * [Steam](http://steampowered.com)
  * [nodejs](http://nodejs.org)
  * [Github client](http://desktop.github.com/)
- * [Sublime Text 3](http://www.sublimetext.com/) **optional**
+ * [Visual Studio Code](http://code.visualstudio.com/) `Text editor which includes support for github, editorconfig and much more`
```
Comment (chrisinajar) [inline docs/install.md]: please remove this, VSC is an absolute garbage fire for dota modding purposes, we do not want to have to answer questions about it. Sublime text supports all of that shit, is much faster, is much more widely used, and we've already made it our standard.  Also this has nothing to do with gulp.

File: docs/install.md [hunk 21]
```diff
@@ -1,51 +1,73 @@
...
+ * `Open Angel arena folder` is a folder name `oaa` inside of `addons folder`, it contains all the files which are present in this github repo 
+
+**`Dota2 addons folder` and `Open Angel arena folder` may not exist for the first time, they will be created after `Prepare Addon Directory` step**
+
+Final structure sholud look like `C:\(some path to steam library)\Steam\steamapps\common\dota 2 beta\addons\oaa\`
+
+### Navigating with command prompt
+Use command `cd` and the undergoing path to navigate,(`cd` works relative to your current path), for example
+```
+dota 2 beta\addons> cd oaa
```
Comment (chrisinajar) [inline docs/install.md]: this step is redundant with the next step, and also that directory wont exist at this point in the documentation.  Also this has nothing to do with gulp.

File: docs/install.md [hunk 22]
```diff
@@ -1,51 +1,73 @@
...
+```
+will go from `Addons` folder to `Open Angel Arena` folder, and
+```
+dota 2 beta\addons\oaa> cd ../
```
Comment (chrisinajar) [inline docs/install.md]: overly complex instructions with more than 2 redundant steps which rely on future steps. Also this has nothing to do with gulp.

File: docs/install.md [hunk 23]
```diff
@@ -1,51 +1,73 @@
...
 ```
-**Note**: The directory path may be different depending on how you installed Steam. If Steam is installed on another drive you'll need to change drives using the letter then a `:` symbol. For example, `E:`.
 
-To ensure the addons directory is set up, run
+### Download and Deploy Project
+To ensure the addons directory is set up, navigate to `Dota2 Installation` folder and run
 ```
 d2am list
 ```
-leave the command prompt open, we'll come back to it. Join us on iscordDay, the URL ends in WNFBB4d. You found the easter egg!
```
Comment (chrisinajar) [inline docs/install.md]: please do not remove this, it's the only way people find the discord right now. Also this has nothing to do with gulp.

File: docs/install.md [hunk 24]
```diff
@@ -1,51 +1,73 @@
...
-
-**You must link the addon before it will work**. You can do this from that same node.js window you had open by running
+This will create `Dota2 addons folder` folder and set it up!
+Now you have to open `GitHub Client` and using top left `+` button `clone` a repo `OAA` to your newly created `Dota2 addons folder`
+To become more familiar with `Git` and `GitHub` you can read a [guide for newcomers](/docs/github-for-noobs.md)
+Now we need to link this project to `Dota 2 Tools` with command
 ```
 d2am link
 ```
-You will see output about creating links for `oaa` game and content.
```
Comment (chrisinajar) [inline docs/install.md]: please do not remove this note, many people needed it to know the command succeeded. Also this has nothing to do with gulp.

File: docs/install.md [hunk 25]
```diff
@@ -1,51 +1,73 @@
...
 ## Launch Tools
-Next we open the actual addon! This is done by right clicking on `Dota 2` from Steam and selecting "Launch Dota 2 - Tools". It should be the second option on the dropdown below "Play Game..."
+Next we open the actual addon! This is done by right clicking on `Dota 2` from Steam and selecting `Launch Dota 2 - Tools`. It should be the second option on the dropdown below `Play Game`
+
+Open `oaa` from the window that opens, and you're in!
+
+The instructions of testing are present in [Next guide](/docs/testing.md)
 
-Open `oaa` from the window that opens, and you're in! Open one of the maps and click the gamepad sybol to test the gamemode. Any changes you make will show up in the Github client, read for a pull request to be made.
+Also all the development is discussed in Discord, so please join us at https://discord.gg/WNFBB4d
```
Comment (chrisinajar) [inline docs/install.md]: See above notes about the discord link. Also this has nothing to do with gulp.

File: docs/russian/github-for-noobs.md [hunk 26]
```diff
@@ -1,173 +0,0 @@
-## Инструкция по работе с сайтом/приложением "GitHub":
```
Comment (chrisinajar) [inline docs/russian/github-for-noobs.md]: Why did you remove the russian documentation? Many people found this very useful. Also this has nothing to do with gulp.

Comment (ProstoSanja) [inline docs/russian/github-for-noobs.md]: There are two files with exactly same contents, didnt you see it?

**Comment by ProstoSanja** [comment]
i tried to make it readable
you didnt like it, i wont rewrite it
